dashboard/src/actions/repos.ts

```diff
@@ -132,68 +130,16 @@ export const installRepo = (
 ): ThunkAction<Promise<boolean>, IStoreState, null, AppReposAction> => {
   return async (dispatch, getState) => {
     try {
-      const {
-        config: { namespace },
-      } = getState();
-      interface ISecretKeyRef {
-        key: string;
-        name: string;
-      }
-      const auth: {
-        header?: { secretKeyRef: ISecretKeyRef };
-        customCA?: { secretKeyRef: ISecretKeyRef };
-      } = {};
-      const secrets: { [s: string]: string } = {};
-      const secretName = `apprepo-${name}-secrets`;
-      if (authHeader.length || customCA.length) {
-        // ensure we can create secrets in the kubeapps namespace
-        if (authHeader.length) {
-          auth.header = {
-            secretKeyRef: {
-              key: "authorizationHeader",
-              name: secretName,
-            },
-          };
-          secrets.authorizationHeader = btoa(authHeader);
-        }
-        if (customCA.length) {
-          auth.customCA = {
-            secretKeyRef: {
-              key: "ca.crt",
-              name: secretName,
-            },
-          };
-          secrets["ca.crt"] = btoa(customCA);
-        }
-      }
-      let syncJobPodTemplateObj = {};
-      if (syncJobPodTemplate.length) {
-        syncJobPodTemplateObj = yaml.safeLoad(syncJobPodTemplate);
-      }
```

Comment:
Frontend no longer needs complication of creating associated secrets etc., since it's done as part of the backend API call implementation.